crates/swc_css_minifier/src/lib.rs

```diff
@@ -3,16 +3,34 @@
 #![allow(clippy::match_like_matches_macro)]
 #![allow(clippy::needless_update)]
 
+use swc_atoms::{js_word, JsWord};
 use swc_css_ast::*;
 use swc_css_visit::VisitMutWith;
 
 use self::{compress::at_rule::compress_at_rule, compressor::compressor, options::MinifyOptions};
 
 mod compress;
 mod compressor;
+mod escape;
 pub mod options;
 
 pub fn minify(stylesheet: &mut Stylesheet, _options: MinifyOptions) {
     stylesheet.visit_mut_with(&mut compressor());
     stylesheet.visit_mut_with(&mut compress_at_rule());
 }
+
+#[inline]
+fn is_css_wide_keywords(ident: &JsWord) -> bool {
+    match ident.to_ascii_lowercase() {
+        js_word!("none")
```

Comment:
This is not CSS-Wide keywords. Ref: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/css/properties/css_parsing_utils.cc;l=3439-3445?q=%20IsCSSWideKeyword&ss=chromium

```c++
// https://drafts.csswg.org/css-values-4/#css-wide-keywords
bool IsCSSWideKeyword(StringView keyword) {
  return EqualIgnoringASCIICase(keyword, "initial") ||
         EqualIgnoringASCIICase(keyword, "inherit") ||
         EqualIgnoringASCIICase(keyword, "unset") ||
         EqualIgnoringASCIICase(keyword, "revert") ||
         EqualIgnoringASCIICase(keyword, "revert-layer");
  // This function should match the overload before it.
}
```

Reply:
Yes, `none` is only for keyframes, so we should not put it here to avoid future problems when we will use `is_css_wide_keyword`